### PR TITLE
 [client/rest]: replace custom uint64 with bigint

### PR DESCRIPTION
**Pull request by Jaguar0625**
problem: REST is using custom uint64 implementation introduced prior to bigint standardization
    solution: replace it with bigint

### CODE REVIEW

**Review by AnthonyLaw — APPROVED** [review]
👍🏼

**Review by AnthonyLaw — APPROVED** [review]

File: client/rest/src/plugins/metadata/metal.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ export const metal = {
 		if (!input.length)
 			throw new Error('input must not be empty');
 		const hashedBytes = sha3_256(input);
-		const uint64 = catapult.utils.uint64.fromBytes(hashedBytes.subarray(0, 8));
-		return [uint64[0], uint64[1] & 0x7FFFFFFF];
+		const uint64 = utils.bytesToBigInt(hashedBytes.subarray(0, 8), 8);
+		return uint64 & 0x7FFFFFFFFFFFFFFFn;
```
Comment (Wayonb) [inline client/rest/src/plugins/metadata/metal.js:37]: can remove from the return message ``, represented as an array of two numbers.``

File: client/rest/src/plugins/mosaic/MosaicDb.js [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default class MosaicDb {
 	 * @returns {Promise<Array<object>>} Mosaics.
 	 */
 	mosaicsByIds(ids) {
-		const mosaicIds = ids.map(id => new Long(id[0], id[1]));
+		const mosaicIds = ids.map(id => Long.fromBigInt(id));
```
Comment (Wayonb) [inline client/rest/src/plugins/mosaic/MosaicDb.js:65]: ``Array<module:utils/uint64~uint64>`` => ``Array<BigInt>`` ?

File: client/rest/src/routes/networkRoutes.js [hunk 3]
```diff
@@ -188,7 +186,7 @@ export default {
 			// rootNamespaceRentalFeePerBlock -> uint64
```
Comment (Wayonb) [inline client/rest/src/routes/networkRoutes.js:186]: uint64 => BigInt?

**Review by Wayonb — APPROVED** [review]
👍🏾

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**